<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>604</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777213</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QLineEdit" name="FindeLineEdit"/>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="FindeButton">
          <property name="text">
           <string>Find</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView">
          <property name="cursor" stdset="0">
           <cursorShape>CrossCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="student">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="Housings">
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <layout class="QGridLayout" name="gridLayout_5">
                   <item row="1" column="0">
                    <widget class="QRadioButton" name="Housing2">
                     <property name="text">
                      <string>Housing2</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <widget class="QRadioButton" name="Housing3">
                     <property name="text">
                      <string>Housing3</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QRadioButton" name="Housing1">
                     <property name="text">
                      <string>Housing1</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="Floors">
                   <property name="title">
                    <string>Floors</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_2">
                    <item row="0" column="0">
                     <layout class="QGridLayout" name="gridLayout">
                      <item row="0" column="0">
                       <widget class="QRadioButton" name="mainFloor">
                        <property name="text">
                         <string>Ground floor</string>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="0">
                       <widget class="QRadioButton" name="plainFloor">
                        <property name="text">
                         <string>Typical floor</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <widget class="QCheckBox" name="showAllCheckBox">
                 <property name="text">
                  <string>Show All   </string>
                 </property>
                </widget>
               </item>
              </layout>
              <zorder></zorder>
              <zorder>showAllCheckBox</zorder>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextBrowser" name="label"/>
          </item>
          <item>
           <widget class="QTextBrowser" name="legend"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="ImageLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1173</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
